bookMyShow/src/utills/helpers.js

```diff
@@ -0,0 +1,21 @@
+const getAllMoviesQuery = (theatreId, city) => {
+  return `SELECT TI.Id, TI.Screen, TI.AVType, TI.SubtitlesLanguage, TI.BookingStatus, TI.ShowTiming, TI.ShowDate, M.MovieName, M.MovieLanguage, M.MovieDimension, M.MovieCertification, TH.City, TH.TheaterName
+  FROM myData.TheatersInfo TI
+  INNER JOIN myData.Movies M
+  ON TI.MovieId = M.MovieId
+  INNER JOIN myData.Theaters TH
+  ON TI.TheaterId = TH.TheaterId
+  WHERE TI.TheaterId = ${theatreId} AND TH.City = '${city}'`;
+};
+
+const getMoviesByDateQuery = (theatreId, city, date) => {
+  return `SELECT TI.Id, TI.Screen, TI.AVType, TI.SubtitlesLanguage, TI.BookingStatus, TI.ShowTiming, TI.ShowDate, M.MovieName, M.MovieLanguage, M.MovieDimension, M.MovieCertification, TH.City, TH.TheaterName
+    FROM myData.TheatersInfo TI
+    INNER JOIN myData.Movies M
+    ON TI.MovieId = M.MovieId
+    INNER JOIN myData.Theaters TH
+    ON TI.TheaterId = TH.TheaterId
+    WHERE TI.TheaterId = ${theatreId} AND TH.City = '${city}' AND TI.ShowDate = '${date}'`;
+};
+
+module.exports = { getAllMoviesQuery, getMoviesByDateQuery };
```

Comment:
We can create a dedicated data layer for this. Better write model classes/functions to define your DB models and Data Transfer Objects. Sequelize is being used but not to its full potential.